### PR TITLE
[fix] Javafx dependency and Travis jdk11

### PR DESCRIPTION
**Pull request by Avanatiker**
**Description**
Add javafx to Gradle dependencies so no manual installation is necessary.

**Describe how this pull is helpful**
Its a nightmare to set up manually.

**Ideas for improvements**
This happens on activating Chams module.

Rendering fonts on 4K looks a bit crumbly

HUD is weired

Font of LagNotifier is broken

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -18,6 +18,13 @@ buildscript {
     }
 }
 
+plugins {
+    id 'application'
+    id 'org.openjfx.javafxplugin' version '0.0.8'
+}
+
+mainClassName = '...'
```
Comment (l1ving) [inline build.gradle:26]: I don't think this is needed if it's empty....

File: build.gradle [hunk 2]
```diff
@@ -18,6 +18,13 @@ buildscript {
     }
 }
 
+plugins {
```
Comment (l1ving) [inline build.gradle:21]: just use the apply plugin below, or move the apply plugins here. consistency is nice

**Review by l1ving — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (Avanatiker) [reply build.gradle:26]: that mainClassName was fixing the client [build]

File: build.gradle (hunk 2 above)
Comment (Avanatiker) [reply build.gradle:21]: how

**Review by l1ving — APPROVED** [review]

**Comment by l1ving** [comment]
I'll have to close this as won't fix. This is incompatible with Oracle JDK (which already has JavaFX) and OpenJDK 8, and most people don't use OpenJDK 11, reviewed this on Discord with you. I'll add this to the docs in the future.

**Pull request closed without merging**